src/test/resources/ReaderServiceImpl/correctinput.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity

```

Comment:
sub-folder names should be lower case
src/test/resources/reader/correctinput.csv